deploy/hooks_conf_only/post-restore

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
```

Comment:
is it possible to remove the hook completely? as it does nothing

Reply:
it is there to overwrite the default system hooks wich raises an error:
`rm /tmp/np` is failing. 
```
$ cat /etc/deploy/hooks/post-restore
#!/bin/sh -e
#
# variables set here:
#   $TARGET: name of the symbolic remote host key (see remote_hosts
#            section in config file)
#

rm /tmp/np

if [ -x /bin/systemctl ]; then
    sudo /bin/systemctl start apache2
elif [ -x /etc/init.d/apache2 ]; then
    sudo /etc/init.d/apache2 start
fi
```